### ISSUE TITLE
Pass pair Class/method instead of closure?

### ISSUE THREAD

**Issue by kopiyka** [bramus/router#17]
Would be nice to have an ability to use Class::classMethod instead of function
$router->get('/', 'Class', 'classMethod');

**Comment by tleb** [comment]
You can already do this.

The class uses the `call_user_func_array`, so you can pass any callable, e.g:

```
<?php

use Bramus\Router\Router;

require 'vendor/autoload.php';

$router = new Router;

// closure
$router->get('/', function () {
    echo 'Foo';
});

// function
$router->get('/', 'function_name');

// method
$router->get('/', ['ClassName', 'methodName']);
```

**Comment by kopiyka** [comment]
Ok, thanks, I'll try

**Comment by vladvisinescu** [comment]
this doesn`t seem to work, this is the error I get:

call_user_func_array() expects parameter 1 to be a valid callback, non-static method PagesController::index()

**Comment by sant0ro** [comment]
@vladvisinescu this method is static. After my commit the Router also has the ability of static calls.
@bramus didn't released the `1.1` version of the router.

But with the actual version you can do the following with static invocations you need pass the Namespace reference. Also you can do "ClassName::StaticMethod" without passing as a Class Callback.

**Comment by bramus** [comment]
Just a quick mention that last week I finally got to tagging 1.1 and releasing it. Extended documentation is still missing though (see https://github.com/bramus/router/issues/25).
